featuresScore.java

```diff
@@ -0,0 +1,76 @@
+
+public class featuresScore {
```

Comment:
Actually, how about we follow the CS2103T coding convention? haha
Link <a href="https://docs.google.com/document/pub?id=1iAESIXM0zSxEa5OY7dFURam_SgLiSMhPQtU0drQagrs&amp&embedded=true">here</a>.